cmake/README.md

```diff
@@ -18,6 +18,7 @@ The following variables are provided for oneDPL configuration:
 | ONEDPL_AOT_ARCH              | STRING | Architecture options for ahead-of-time compilation, supported values can be found [here](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-dpcpp-cpp-compiler-dev-guide-and-reference/top/compilation/ahead-of-time-compilation.html)                                                                                            | "*" for GPU device and "avx" for CPU device |
 | ONEDPL_TEST_EXPLICIT_KERNEL_NAMES   | STRING | Control kernel naming. Affects only oneDPL test targets. Supported values: AUTO, ALWAYS. AUTO: rely on the compiler if "Unnamed SYCL lambda kernels" feature is on, otherwise provide kernel names explicitly; ALWAYS: provide kernel names explicitly | AUTO          |
 | ONEDPL_TEST_WIN_ICX_FIXES     | BOOL   | Affects only oneDPL test targets.  Enable icx, icx-cl workarounds to fix issues in CMake for Windows.                      | ON            |
+| ONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION | BOOL | Avoid using known identity for 64-bit arithmetic data types                       | UNDEFINED     |
```

Comment:
@dmitriy-sobolev which proper type for this option `ONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION` we should declare?

Reply:
"Known identity" looks like an implementation detail, it may confuse a user, because the consequences are not apparent.
What do you think about adding a line into the description which would highlight/imply a consequence of using this workaround and refer known issues section?

I suggest something like: "Fall back to more general, usually less optimized, implementations of reduction-based algorithms for 64-bit types. Refer to 'Known issues' section for more details".

Reply:
This proposal has been applied.
@akukanov Do you like this text?

Reply:
Sorry for not answering the original question. I thought about a default value, rather than a type.

I am fine with `BOOL` since there is a precendent with `ONEDPL_USE_UNNAMED_LAMBDA`, which can also have "true", "false" and "non-defined" states. I am a bit confused since it can accept any value (it is not limited to "bool"), but I am not aware of a better alternative.